### PR TITLE
Remove 2023.02 (bobcat) from COU

### PR DESCRIPTION
**Pull request by chanchiwai-ray**

### CODE REVIEW

**Review by jneo8 — DISMISSED** [review]

File: tests/unit/utils/test_openstack.py [hunk 1]
```diff
@@ -289,7 +282,6 @@ def test_openstack_release_setter():
 def test_openstack_release_setter_by_date():
     openstack_release = OpenStackRelease("2023.1")
     assert openstack_release.codename == "antelope"
-    assert openstack_release.next_release == "bobcat"
```
Comment (samuelallan72) [inline tests/unit/utils/test_openstack.py:292]: Can we assert that the next release is caracal here?

Comment (chanchiwai-ray) [reply tests/unit/utils/test_openstack.py:292]: Thanks, it's now tested in `test_plan.py`

**Review by jneo8 — APPROVED** [review]

**Review by samuelallan72 — APPROVED** [review]
Thanks, looks good :D

**Comment by gabrielcocenza** [comment]
I just had time to see this now. I'm not completely sure that this is the right approach once the community as a general thing can have a cloud running on bobcat and then COU will not support upgrading it. IMO the ideal scenario would be:

- if COU detect a cloud on Antelop, upgrade directly to Caracal
- if COU detect a cloud on Bobcat, upgrade to Caracal or in the worst scenario identify that the cloud is on Bobcat and say that is not supported and manual intervention is needed

If I understood this correctly COU will break if a Bobcat cloud is passed right?

cc: @aieri

**Comment by samuelallan72** [comment]
@gabrielcocenza we've discussed this in APAC and the conclusion has been that we simply will not support a cloud on Bobcat.  Afaik this means worst case it will crash or have undefined operation, best case is identify Bobcat and stop with a message about unsupported cloud version.

**Comment by chanchiwai-ray** [comment]
Thanks @samuelallan72 . 

@gabrielcocenza Yes, this is raised by me already, and we've confirmed that we (APAC) accept the risk. Please refer to the internal ticket for context.

**Pull request merged**